akka-stream/src/main/scala/akka/stream/impl/ActorMaterializerImpl.scala

```diff
@@ -36,6 +36,14 @@ abstract class ExtendedActorMaterializer extends ActorMaterializer {
     subflowFuser:      GraphInterpreterShell ⇒ ActorRef,
     initialAttributes: Attributes): Mat
 
+  // TODO not sure if needed exposing, I think so, at least right now -- ktoso
+  private[akka] def materialize[Mat](
```

Comment:
please sanity check if exposing like this is ok, I need to call it with my replaced phase.